src/cdk/tree/tree.ts

```diff
@@ -363,10 +362,10 @@ export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
   focus(): void {
     this._elementRef.nativeElement.focus();
   }
-
+  
   protected _setRoleFromData(): void {
     if (this._tree.treeControl.isExpandable) {
-      this.role = this._tree.treeControl.isExpandable(this._data) ? 'group' : 'treeitem';
+      this.role = this._tree.treeControl.isExpandable(this._data) ? 'treeitem' : 'treeitem';
```

Comment:
@annieyw This condition outputs the same in either case, should this just be `this.role = 'treeitem'`?